Cargo.toml

```diff
@@ -45,6 +45,15 @@ lazy_static = "1.4.0"
 once_cell = "1.10.0"
 strum = "0.24"
 strum_macros = "0.24"
+derive-getters = "0.2.0"
+
+[dependencies.uuid]
```

Comment:
introduce `uuid` for generate unique `buildInvocationId`. As it is optional, if we left it empty, can be removed.